src/InstallUpgradeBC/CAVSBLicensingInstall.Codeunit.al

```diff
@@ -0,0 +1,50 @@
+codeunit 71264333 "CAVSB Licensing Install"
+{
+    Subtype = Install;
+
+    var
+        CavalloProfitScanSubscriptionIdTok: Label '6fa64f0a-fad4-4d5a-8c84-b2de9fd96fa8';
+
+    trigger OnInstallAppPerDatabase()
+    var
+        CAVSBTelemetry: Codeunit "CAVSB Telemetry";
+    begin
+        PerformInstallOfTestSubscriptions();
+        CAVSBTelemetry.LicensingAppInstalled();
+    end;
+
+    procedure PerformInstallOfTestSubscriptions()
+    begin
+        AddTestProduct(Enum::"CAVSB License Platform"::CavalloProfitScan, CavalloProfitScanSubscriptionIdTok);
```

Comment:
🔧 Remove this, I think we will have each of our extensions add this in their own migration with their own implementations